pom.xml

```diff
@@ -48,6 +48,19 @@
             <version>2.3.4.RELEASE</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>2.9.2</version>
```

Comment:
I want to manage versions in single place in pom
please take a look how to handle dependency versions in pom